src/detail/Platform.cpp

```diff
@@ -51,11 +51,10 @@ std::string demangle(const char *name) {
 
 Maybe<std::string> getEnvValue(const std::string &name) {
   char *buffer = nullptr;
-  const auto ret = _dupenv_s(&buffer, nullptr, name.c_str());
-  const auto ptr = std::unique_ptr<char, decltype(&std::free)>(buffer, &std::free);
+  _dupenv_s(&buffer, nullptr, name.c_str());
```

Comment:
According to https://msdn.microsoft.com/en-us/library/ms175774.aspx
```
It is the calling program's responsibility to free the memory by calling free.
```

By removing the `unique_ptr`, you're also removing a call to `std::free`.